README.md

```diff
@@ -1,24 +1,59 @@
-# 👾 Processbot
+# 👾 processbot
 
 # Commands
 
 To be posted as a comment in the Pull Request. **Your whole comment should only have the command**.
 
-- `bot merge` to automatically merge it once checks pass (if approvals have been
-  given).
-- `bot merge force` to attempt merge without waiting for checks (if approvals
-  have been given).
-- `bot merge cancel` to cancel a pending `bot merge`.
-- `bot compare substrate` to see a diff between current branch's Substrate.
+- `bot merge`: [if approved](#criteria-for-merge), merge once checks pass.
+- `bot merge force`: [if approved](#criteria-for-merge), merge immediately
+  while disregarding checks.
+- `bot merge cancel`: cancel a pending `bot merge`.
+- `bot compare substrate`: see a diff between current branch's Substrate
   version and the latest Polkadot release's Substrate version.
-- `bot rebase` to merge origin/master.
-- `bot burnin` to build and deploy the PR for a burn-in test.
+- `bot rebase`: merge origin/master into the PR.
```

Comment:
1. Does it create a commit message?
2. If yes, it may start a pipeline, does it?
3. If yes, is it already a part of "pre-merge pipelines" feature request? (because if not, you could safely add `[skip ci]` to the commit message to avoid spamming pipelines)

Reply:
It does create a commit. The point 2. you mentioned is implied by this because `[skip ci]` is not being used. After a master merge, the build might fail due to no "pre-merge pipelines" so it's relevant for CI not to be skipped IMO.

Even after pre-merge pipelines gets implemented, this has a different purpose from what I see. Someone might want to catch up with master before their MR is ready to merge (during development). It was used semi-recently in https://github.com/paritytech/polkadot/pull/2858#issuecomment-816353954 so I suppose it's somewhat useful.

Reply:
> Even after pre-merge pipelines gets implemented, this has a different purpose from what I see.

Yes, you're right. This was requested solely to optimize doing this manually AFAIR.